packages/replay-next/components/sources/log-point-panel/LogPointPanel.tsx

```diff
@@ -340,7 +348,11 @@ function PointPanelWithHitPoints({
       <div className={styles.EditableContentWrapperRow}>
         {showTooManyPointsMessage ? (
           <div className={styles.ContentWrapperTooManyPoints}>
-            Use Focus Mode to reduce the number of hits.
+            Use{""}
+            <span style={{ textDecoration: "underline" }} onClick={enterFocusMode}>
```

Comment:
For consistency with the rest of the app, styles like this should be in the CSS module, not inline. (We only use inline styles for _dynamic_ things.)

Reply:
Ah ha, that makes sense. Thank you, will address!

Reply:
That would be my fault! :)